tests/acceptance/features/apiAntivirus/antivirus.feature

```diff
@@ -127,3 +127,37 @@ Feature: antivirus
       | old              |
       | new              |
       | spaces           |
+
+
+  Scenario: upload a file with virus to a spaces inside the folder
+    Given using spaces DAV path
+    And the administrator has given "Alice" the role "Space Admin" using the settings api
+    And user "Alice" has created a space "new-space" with the default quota using the GraphApi
+    And user "Alice" has created a folder "uploadFolder" in space "new-space"
+    When user "Alice" uploads a file inside space "new-space" with content "X5O!P%@AP[4\PZX54(P^)7CC)7}$EICAR-STANDARD-ANTIVIRUS-TEST-FILE!$H+H*" to "/uploadFolder/virusFile.txt" using the WebDAV API
+    Then the HTTP status code should be "201"
+    And user "Alice" should get a notification with subject "Virus found" and message:
+      | message                                                                        |
+      | Virus found in virusFile.txt. Upload not possible. Virus: Win.Test.EICAR_HDB-1 |
```

Comment:
```suggestion
    When user "Alice" uploads a file inside space "new-space" with content "X5O!P%@AP[4\PZX54(P^)7CC)7}$EICAR-STANDARD-ANTIVIRUS-TEST-FILE!$H+H*" to "/virusFile.txt" using the WebDAV API
    Then the HTTP status code should be "201"
    And user "Alice" should get a notification with subject "Virus found" and message:
      | message                                                                        |
      | Virus found in virusFile.txt. Upload not possible. Virus: Win.Test.EICAR_HDB-1 |
```

Reply:
The scenario says `Scenario: upload a file with virus to a spaces inside the folder`.

Reply:
we can change the scenario to `Scenario: upload a file with a virus to a project space` .  Try to upload a file with virus in project space root level and inside the folder as well. I don't think we  need to write a separate scenario for this one